src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -10805,5 +10805,44 @@ public async Task FormatRawStringInterpolation2()
 var s = $""""""{s, 0 : x }""""""
 ");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        public async Task AnonymousMethodIntendBraces_01()
+        {
+            var options = new OptionsCollection(LanguageNames.CSharp)
+            {
+                { IndentBraces, true },
+                { NewLinesForBracesInAnonymousMethods, true }
```

Comment:
would love to see these tests validate all 4 combinations of values.  you can use a theory for this if you woudl like.  the same holds for the other test.